python/cudnn/sdpa/fwd/kernels/prefill_d192_d128_mxfp8_sm100.py

```
@@ -3011,41 +3021,58 @@ def _correction_warp_group(
+            if cutlass.const_expr(_FP32_PARTIALS):
+                # fp32 partials: the accumulator goes straight to the workspace,
+                # bypassing the SMEM O tile and its TMA store.
+                _store_fp32_partial_tile(
+                    o_partial_f32,
+                    tmem_base_epi,
+                    tmem_O_off,
+                    inv_sum,
+                    _row_valid,
+                    _partial_batch(batch_idx, split_idx, n_batch),
+                    q_row_global,
+                    row_head_idx,
```

Ruff (0.16.3)

[error] Undefined name `row_head_idx`

(F821)